packages/ffe-cards-react/src/index.d.ts

```diff
@@ -30,19 +30,19 @@ export interface CardRenderProps {
     Text: React.FC<TextProps>;
 }
 
-export interface ImageCardProps extends CardBaseProps {
+export interface ImageCardProps extends Omit<CardBaseProps, 'children'> {
```

Comment:
vad med og endre Children her istellet?

```
export interface ComponentBaseProps
    extends React.HtmlHTMLAttributes<
        HTMLElement | HTMLButtonElement | HTMLAnchorElement
    > {
    className?: string;
    element?: HTMLElement | string | React.ElementType;
    children?: React.ReactNode | ((props: CardRenderProps) => React.ReactNode);
}

````

Reply:
I react-koden er begge `ComponentBase.propTypes.children` og `CardBase.propTypes.children` lik `PropType.node` og det skulle bli en missmatch.
Men i og for seg så føler jeg at slik det er nå og alle interfaces arver fra `ComponetBaseProps` blir litt rotet fordi det ikke tilsvarer semantikken i react koden :thinking: